pkg/metadata/metadata.go

```diff
@@ -51,9 +51,9 @@ type TableMetadata struct {
 }
 
 type Part struct {
-	Partition string `json:"partition,omitempty"`
-	Name      string `json:"name"`
-	Required  bool   `json:"required,omitempty"`
+	Partition      string `json:"partition,omitempty"`
+	Name           string `json:"name"`
+	DuplicatedPart bool   `json:"required,omitempty"`
```

Comment:
Sorry, I can't approve this PR
I undestand maybe  "requiered" is not so mental name as you expected in the code, but differences between `json` and internal variable name is worst variant.

Reply:
> Sorry, I can't approve this PR I undestand maybe "requiered" is not so mental name as you expected in the code, but differences between `json` and internal variable name is worst variant.

What does this "required" mean?

Reply:
For incremental backup, we have the parent backup which marked as `required` in backup_name/metadata.json
and some parts are not present in incremental backup and present only in `required` backup, so it also marked as `required`

During download incremental backup, `required` backup downloads first and during unpack `required` parts create as hard links.